### DISCUSSION TITLE
DataTables support?

### DISCUSSION THREAD

**Discussion by rgbinnovation** [themeselection/flyonui#10] [Ideas / Request new feature]
Any plans to include skinned DataTables or other similar solution? Seems to the only thing lacking?

**Comment by PruthviPraj00** [comment]
Hello @rgbinnovation,  

We'll be adding DataTables in the near future. Please stay tuned!

Regards

**Comment by PruthviPraj00** [comment]
Hello @rgbinnovation,  

The new FlyonUI version 1.2.0 now includes a Datatable feature!  
You can learn more about it here: [[FlyonUI Datatable](https://flyonui.com/docs/third-party-plugins/datatables/)](https://flyonui.com/docs/third-party-plugins/datatables/).
